resources/views/account/partials/layout.blade.php

```diff
@@ -1,17 +1,32 @@
 @extends('rapidez::layouts.app')
 
 @section('title', __('Account'))
+@php
+    $showNavigation = !request()->is('account');
+    $backurl = match (true) {
+        request()->is('account') => null,
+        request()->is('account/order/*') => '/account/orders',
+        default => '/account',
+    };
+@endphp
 
 @section('content')
-    <div class="container" v-cloak>
+    <div
+        class="container"
+        v-cloak
+    >
         <template v-if="$root.user?.id">
```

Comment:
We've this now: https://github.com/rapidez/core/commit/dc1b5242926145bbb55552b679df5a081b84d434